core/include/webview/detail/backends/cocoa_webkit.hh

```diff
@@ -628,12 +573,12 @@ private:
     objc::msg_send<void>(m_widget, "addSubview:"_sel, m_webview);
     objc::msg_send<void>(m_webview, "setFrame:"_sel,
                          objc::msg_send_stret<CGRect>(m_widget, "bounds"_sel));
+    objc::msg_send<void>(m_window, "setContentView:"_sel, m_widget);
```

Comment:
This line should be part of the window setup, not the widget setup.

Reply:
Fixed in latest commit